scalecube-configuration/src/main/java/io/scalecube/configuration/RequestValidator.java

```diff
@@ -56,6 +66,9 @@ private static void validateRepository(String repository) {
     if (repository == null || repository.trim().isEmpty()) {
       throw new IllegalArgumentException("Please specify 'repository'");
     }
+    if (repository.contains("::")) {
```

Comment:
Maybe should use regex, because I can specify a repository name as `bla:::bla`.

Reply:
::: contains ::